app/Console/Commands/Wiki/OrgDigitalBibleLibraryCompare.php

```diff
@@ -83,7 +83,11 @@ private function organizationExists($dbl_org)
     private function fuzzySearchOrgs($dbl_org)
     {
         // Otherwise Fuzzy Search for Provider Name
-        $organizations = Searchy::driver('ufuzzy')->search(config('database.connections.dbp.database').'.organization_translations')->fields('name')->query($dbl_org->full_name)->getQuery()->limit(5)->get();
+        // $organizations = Searchy::driver('ufuzzy')->search(config('database.connections.dbp.database').'.organization_translations')->fields('name')->query($dbl_org->full_name)->getQuery()->limit(5)->get();
+        $organizatios = OrganizationTranslation::whereFuzzy('name', $dbl_org->full_name)
```

Comment:
typo. should be organizations

Reply:
It has been fixed in the next PR:
- https://github.com/faithcomesbyhearing/dbp/pull/474